### ISSUE TITLE
Background scaling problem with Nativescript 4.2.0

### ISSUE THREAD

**Issue by FranciZ** [NativeScript/NativeScript#5905]
### Tell us about the problem
The CSS background sizing does not seem to work correctly on pre Oreo Android devices.

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:
- CLI: 4.2.0-2018-05-30-11723
- Cross-platform modules: tns-core-modules@4.0.1
- Runtime(s): 4.2.0-2018-05-31-01

### Please tell us how to recreate the issue in as much detail as possible.
The below code would correctly scale the image in Nativescript 3.4.2 on all versions of Android but in 4.2.0 it distorts and fits the image to fit the element it's displayed in. It works correctly in Android 8.1.0 but distorts in pre Android 8 devices as far as I can tell.

```html
<ContentView style="background-position: center; background-size: cover;"
                 [style.backgroundImage]="mapBackground"
                 height="100%"
                 width="100%"
                 top="0"
                 left="0"
                 [isUserInteractionEnabled]="false">
```

### Image comparison

Both the map background and the logo are distorted. On the older Samsung phone.

![image](https://user-images.githubusercontent.com/2536035/40871419-9170084c-663b-11e8-84c6-628da0b3e6c7.jpg)

**Comment by tsonevn** [comment]
Hi @FranciZ,
Can you provide more info about the problem and about the API level of the older device, which you used for testing? It will help if you provide sample project, which demonstrates the issue and can be used for debugging.

**Comment by NickIliev** [comment]
@FranciZ I've tried to reproduce the issue using [this test project](https://github.com/NickIliev/NS-Issues-2018-II/tree/master/NativeScript/issue-5905) but the photo seems to load as expected.

**Comment by tralves** [comment]
Hi!
I was having the same issue and, meanwhile, I discovered that it is related to the image size.

I created the project [android-res-image-cover-issue](https://github.com/tralves/android-res-image-cover-issue) to show the issue.

This is the result:
<img width="200" alt="test" src="https://user-images.githubusercontent.com/2600867/48318200-9aff3e80-e5f4-11e8-8109-2acb36f0e3cc.png">

In this project, I added an image resource created from 3 sizes:
huge: 3156 x 2106
medium: 720 x 480
small: 100 x 67

As you see, the huge image is distorted. This only happens on Android. Could be related to some sort of memory issue in Android.

**Comment by tralves** [comment]
Hi!
I was having the same issue and, meanwhile, I discovered that it is related to the image size.

I created the project [android-res-image-cover-issue](https://github.com/tralves/android-res-image-cover-issue) to show the issue.

This is the result:
<img width="200" alt="test" src="https://user-images.githubusercontent.com/2600867/48318200-9aff3e80-e5f4-11e8-8109-2acb36f0e3cc.png">

In this project, I added an image resource created from 3 sizes:
huge: 3156 x 2106
medium: 720 x 480
small: 100 x 67

As you see, the huge image is distorted. This only happens on Android. Could be related to some sort of memory issue in Android.
